internal/pkg/manifest/svc.go

```diff
@@ -441,6 +441,14 @@ func (s *stringOrFromCFN) UnmarshalYAML(value *yaml.Node) error {
 	return nil
 }
 
+// PlainTostringOrFromCFN returns the new stringOrFromCFN struct with the
+// underlying plain field set, holding value.
+func PlainTostringOrFromCFN(value string) *stringOrFromCFN {
```

Comment:
Do we need this func defined here? Public function with private struct output looks odd, and it seems like this is only used in the stack pkg unit tests.

Reply:
Agreed. But I could not think of other way to provide a `FileSystemID` in the stack pkg unit tests as `stringOrFromCFN` is not exportable in other pkg.